ConsumerEdition/guides/multi-usb-camera.md

```diff
@@ -57,6 +56,10 @@ $ sudo apt install vlc
 
 - The theoretical bandwidth of a USB 2.0 Hub is 480mbps or ~60MBps. Adding a USB HUB divides that bandwidth across our three cameras giving us approximately ~20MBps per camera. S the trick is to make sure we are well within that limit.
 
+- Although this guide follows ```vlc```, the concept can be applied to any other software using v4l2 driver.
+
+- Majority of the webcams are compatible with this guide, although there are some webcams that do not play well with v4l2 and will send uncompressed data stream irrespective of the command. Such webcams will not work with this guide. More info on this bug, where cameras over request bandwidth can be found here: http://www.ideasonboard.org/uvc/faq/#faq7
```

Comment:
Do they really *send* uncompressed data? The issue described in the FAQ is that they *allocate* bandwidth for large uncompressed frames regardless of what data they actually transmit (the quirk workaround functions by ignoring how much bandwidth the camera says it wants and letting the kernel make an estimate based on the camera parameters).

Otherwise it looks OK.

Reply:
ok, i'll change that